src/lib_ccx/dvb_subtitle_decoder.c

```diff
@@ -1551,6 +1551,9 @@ static int write_dvb_sub(struct lib_cc_decode *dec_ctx, struct cc_subtitle *sub)
 				rect->ocr_text = ocr_str;
 			else
 				rect->ocr_text = NULL;
+            if (ccx_options.dvb_debug_traces_to_stdout) {
+                mprint("\nOCR Result: %s\n", rect->ocr_text ? rect->ocr_text : "NULL");
```

Comment:
Same thing... plus debug options are controlled with a mask, not with independent variables.

Reply:
I did not add the option. It is controlled by parameter `-debugdvbsub` and is used many places in the code already.